src/test/java/com/datadog/api/RecorderSteps.java

```diff
@@ -0,0 +1,133 @@
+package com.datadog.api;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.recordSpec;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Comparator;
+import java.util.List;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.core.WireMockApp;
+import com.github.tomakehurst.wiremock.recording.RecordSpec;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.When;
+
+public class RecorderSteps {
+
+    protected static String cassettesDir = "src/test/resources/scenarios/cassettes";
+
+    public static final String host = "localhost";
+    public static final int port = 18080 + Integer.parseInt(System.getProperty("surefireForkNumber"));
+
+    private WireMockServer server;
+
+    private World world;
+
+    public RecorderSteps(World world) {
+        this.world = world;
+    }
+
+    public static String getUrl() {
+        return "http://" + host + ":" + port;
+    }
+
+    @Before(order = 1)
+    public void setupClock() throws java.io.IOException {
```

Comment:
It'd be. nice to reuse the code in the current test utils, this looks a lot like the same logic.

Reply:
It is not easy to reuse since the path generation is different.